PlayerController.cs

```diff
@@ -18,19 +23,19 @@ void Update()
         
     }
 
-    [Header("General Setup Settings")]
-    [SerializeField] private InputAction movement;
```

Comment:
This defines Unity's new [Input Management System](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.7/manual/index.html)